app/controllers/admin/sites_controller.rb

```diff
@@ -58,7 +58,7 @@ def set_site
     end
 
     def set_variables_for_sites_neighbourhoods_selection
-      @all_neighbourhoods = policy_scope(Neighbourhood).order(:name)
+      @all_neighbourhoods = policy_scope(Neighbourhood).order(:name).where(unit: 'ward')
```

Comment:
@alexandria-gfsc This is the one we talked about. Maybe all neighbourhoods should not filter for wards, but instead have another function that returns only wards - add params to this one (maybe even default to [ward]) that takes a list of units to scope the query?

Basically, it doesn't seem intuitive to call it all neighbourhoods but filter some out.